ui/src/main/webapp/config/webpack.test.js

```diff
@@ -44,6 +45,11 @@ module.exports = {
             /angular(\\|\/)core(\\|\/)(esm(\\|\/)src|src)(\\|\/)linker/,
             helpers.root('./src'), // location of your src
             {} // a map of your routes
-        )
+        ),
+        new DefinePlugin({
+            'process.env': {
+                ENV: JSON.stringify('test')
```

Comment:
This is stupid, but passing simple string doesn't work... Based on http://stackoverflow.com/questions/35481068/whats-the-proper-way-to-set-node-env-in-karma-for-running-webpack